src/ontology/mondo-ingest.Makefile

```diff
@@ -189,8 +189,9 @@ $(TMPDIR)/component-%.json: $(COMPONENTSDIR)/%.owl
 	$(ROBOT) convert -i $< -f json -o $@
 .PRECIOUS: $(TMPDIR)/component-%.json
 
-$(MAPPINGSDIR)/%.sssom.tsv: $(TMPDIR)/component-%.json metadata/mondo.sssom.config.yml
-	sssom parse $< -I obographs-json --prefix-map-mode merged -m metadata/mondo.sssom.config.yml -o $@
+$(MAPPINGSDIR)/%.sssom.tsv:
+	make $(TMPDIR)/component-$*.json metadata/mondo.sssom.config.yml
```

Comment:
@matentzn Why not dependencies?

The only reason I can remember why we move these goals from dependencies into the recipe block is cases of `.PHONY` goals, to avoid the goal from running 100% of the time. But these aren't phony. Why the move?

Reply:
I realised the dependencies would fire in cases where the mapping is manually curated (i.e. just uploaded, not extracted from a JSON file). This can be seen with the `nando` example in this PR. So `make` tries to generate the json dependency when building `$(MAPPINGSDIR)/mondo-nando.sssom.tsv` but cant, because its not there (and it is not needed).